### PR TITLE
Add 1.1 changelog

### PR DESCRIPTION
**Pull request by mightybyte**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,54 @@
...
+
+* Refactor Pact service state handling and chain data.
+  [#767](https://github.com/kadena-io/chainweb-node/pull/767)
+
+* Fix bug in miner redeem on failed payloads.
+  [#773](https://github.com/kadena-io/chainweb-node/pull/773)
+
+* Set default block gas limit of 6000.
+  [#776](https://github.com/kadena-io/chainweb-node/pull/776)
+
```
Comment (emilypi) [inline CHANGELOG.md]: * Tx compilation failure messages from mempool: 

face77b7cf9447e063413333e4ed691993c58414

* Performance improvements to tx execution by precompiling gas buy/redeem/coinbase phases

92712380365ce41e653b8a83f9e5b47ce84e5026

* Module/Interface install has been disabled

555496f1b104b4ebf7396354d62d922ec9d48944

Comment (mightybyte) [reply CHANGELOG.md]: I lumped 766, 768, and 771 all under the "staged transaction rollout" bullet.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,54 @@
...
+* Improve cut DB membership queries.
+  [#756](https://github.com/kadena-io/chainweb-node/pull/756)
+
+* Better error handling for /mining/solved endpoint.
+  [#762](https://github.com/kadena-io/chainweb-node/pull/762)
+
+* Increase mempool session timeout.
+  [#761](https://github.com/kadena-io/chainweb-node/pull/761)
+
+* Mempool performance improvements
```
Comment (fosskers) [inline CHANGELOG.md]:
```suggestion
* Mempool performance improvements.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,56 @@
 # `chainweb-node` Changelog
 
+## 1.1 (2019-12-02)
+
+This is a major release that turns on coin transfers. All node operators should
```
Comment (fosskers) [inline CHANGELOG.md]:
```suggestion
This is a major release that activates coin transfers. All node operators should
```

File: CHANGELOG.md (hunk 1 above)
Comment (slpopejoy) [reply CHANGELOG.md]: Added some dedicated ones back

**Pull request merged**